Doc/library/datetime.rst

```diff
@@ -934,6 +934,10 @@ Other constructors, all class methods:
 
    This function is preferred over :meth:`today` and :meth:`utcnow`.
 
+   .. note::
+
+      It's possible that subsequent ``datetime.now()`` return the same instant,
+      depending on the precision of the underlying clock.
```

Comment:
How about:
Depending on the precision of the underlying clock, it's possible that subsequent datetime.now() statements return the same instant.